### PR TITLE
feat: sparkles: Add simple Koa.js server with .env and tsx support 

### PR DESCRIPTION
**Pull request by OtavioSC**
A "hello world' route was added to koa.js server with dotenv configuration and tsx script called "dev".

### CODE REVIEW

**Comment by OtavioSC** [comment]
#7

File: apps/server/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "server",
```
Comment (vinibgoulart) [inline apps/server/package.json:2]: use "@project-name/package-name" to name a package. ex: @crud-beer/server

File: apps/server/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "version": "1.0.0",
+  "main": "index.js",
```
Comment (vinibgoulart) [inline apps/server/package.json:4]: index has the extension .ts

**Review by vinibgoulart — COMMENTED** [review]
nice

File: apps/server/package.json (hunk 2 above)
Comment (OtavioSC) [reply apps/server/package.json]: Fixed

File: apps/server/package.json (hunk 1 above)
Comment (OtavioSC) [reply apps/server/package.json]: Fixed

**Review by vinibgoulart — APPROVED** [review]

**Review by daniloab — COMMENTED** [review]
nice!

**Pull request merged**